editor.planx.uk/src/routes/authenticated.tsx

```diff
@@ -68,6 +69,48 @@ const editorRoutes = compose(
       });
     }),
 
+    "/admin-panel": map(async (req) => {
+      const isAuthorised = useStore.getState().user?.isPlatformAdmin;
+      if (!isAuthorised)
+        throw new NotFoundError(
+          `User does not have access to ${req.originalUrl}`,
+        );
```

Comment:
Followup PR: the `/admin-panel` page is successfully reachable from the index, but throws this error if you hard-refresh form the page itself. This also appears to be a current bug with `/global-settings`, so not introduced here. Since these routes are only accessible by platform admins to begin with, there shouldn't be much harm in merging & pushing an overall fix as a followup.